drivers/focuser/sestosenso2.cpp

```diff
@@ -1046,13 +1052,16 @@ IPState SestoSenso2::MoveAbsFocuser(uint32_t targetTicks)
         char res[SESTO_LEN] = {0};
         uint32_t currentTicks = static_cast<uint32_t>(FocusAbsPosN[0].value);
         uint32_t backlash = static_cast<uint32_t>(backlashTicks);
-        if(targetTicks < currentTicks) {
-            if (command->go(currentTicks+backlash, res) == false)
-                return IPS_ALERT;
-            targetTicks -= backlash;
-        } else {
-            if (command->go(currentTicks-backlash, res) == false)
-            targetTicks += backlash;
+        startPos = static_cast<int32_t>(currentTicks);
+        if(backlashDirection != (targetTicks < currentTicks)) {
+            backlashDirection = (targetTicks < currentTicks);
+            if(backlashDirection) {
+                targetTicks -= backlash;
+                backlashDeadzone = static_cast<int32_t>(backlash);
+            } else {
+                targetTicks += backlash;
+                backlashDeadzone = -static_cast<int32_t>(backlash);
+            }
         }
```

Comment:
Does the Backlash function only happen on the FIRST reverse direction of movement?  It looks like Backlash is added to all moves.  Backlash should only be added to the first reverse direction of movement.

Maybe I am missing something?